### PR TITLE
Add client and source tags to resource group documentation

### PR DESCRIPTION
**Pull request by Jessie212**
## Description

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement.

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Addition to documentation.

> How would you describe this change to a non-technical end user or system administrator?

Add client and source tags to resource group documentation

## Documentation

( ) No documentation is needed.
(x) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

### CODE REVIEW

File: docs/src/main/sphinx/admin/resource-groups.rst [hunk 1]
```diff
@@ -198,13 +198,21 @@ The source name can be set as follows:
 
 * CLI: use the ``--source`` option.
 
-* JDBC: set the ``ApplicationName`` client info property on the ``Connection`` instance.
+* JDBC: set the ``ApplicationName`` client info property on the ``Connection``
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/resource-groups.rst:201]: This is for Java programming using the JDBC drvier.

File: docs/src/main/sphinx/admin/resource-groups.rst [hunk 2]
```diff
@@ -198,13 +198,21 @@ The source name can be set as follows:
...
+  instance.
+
+* Driver properties: add the ``source`` tag when using a Java application that
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/resource-groups.rst:204]: These are the JDBC driver properties to use in applications using the driver. Add this as a third bullet in the list and maybe even link to the JDBC driver properties table.

File: docs/src/main/sphinx/admin/resource-groups.rst [hunk 3]
```diff
@@ -198,13 +198,21 @@ The source name can be set as follows:
...
 
 Client tags can be set as follows:
 
 * CLI: use the ``--client-tags`` option.
 
-* JDBC: set the ``ClientTags`` client info property on the ``Connection`` instance.
+* JDBC: set the ``ClientTags`` client info property on the ``Connection``
+  instance.
+
+* Driver properties: add the ``client`` tag when using a Java application that
```
Comment (mosabua) [inline docs/src/main/sphinx/admin/resource-groups.rst:214]: Same as above .. and check the correct property name..

**Review by mosabua — CHANGES_REQUESTED** [review]
Needs a bunch of updates..

**Review by mosabua — APPROVED** [review]

**Review by hashhar — APPROVED** [review]
LGTM % nit

nit: `JDBC used in` -> `JDBC driver as used in`/`JDBC driver when used in/with`?

**Pull request merged**